mods/ra/rules/world.yaml

```diff
@@ -265,6 +265,14 @@ World:
 		PanelName: SKIRMISH_STATS
 	LoadWidgetAtGameStart:
 	ScriptTriggers:
+	TimeLimitManager:
+		TimeLimitWarnings:
+			10: TenMinutesRemaining
```

Comment:
Why didn't you add the notifications for `Twenty`, `Thirty` and `FourtyMinutesRemaining` here? We would need to repeat this entire list in each mission that wants to use those.

Reply:
IMHO having those additional notifications would be distracting, as they'd most probably happen towards the start of the game, not towards its end where you'd expect them. Added them now nonetheless.